pennylane/labs/resource_estimation/ops/op_math/symbolic.py

```diff
@@ -133,6 +521,32 @@ def controlled_resource_decomp(
         num_ctrl_values,
         num_work_wires,
     ) -> Dict[re.CompressedResourceOp, int]:
+        r"""Returns a dictionary representing the resources for a controlled version of the operator.
+
+        Args:
+            outer_num_ctrl_wires (int): The number of control qubits to further control the base
+                controlled operation upon.
+            outer_num_ctrl_values (int): The subset of those control qubits, which further control
```

Comment:
Same comments about using the word "off" to mean when the state is |0>

Reply:
fixed